atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowQueryFactory.java

```diff
@@ -62,27 +61,31 @@ public boolean hasOverflowValues() {
         case FINISHED:
             return ImmutableList.of(getNewOverflowQuery(arg));
         default:
-            throw new EnumConstantNotPresentException(OverflowMigrationState.class, migrationState.name());
+            throw new EnumConstantNotPresentException(OverflowMigrationState.class, config.overflowMigrationState().name());
         }
     }
 
     private FullQuery getOldOverflowQuery(ArrayHandler arg) {
         String query =
-                " /*SQL_MET_SELECT_OVERFLOW */ " +
-                " SELECT /*+ USE_NL(t o) LEADING(t o) INDEX(o pk_pt_metropolis_overflow) */ " +
+                " /* SELECT_OVERFLOW */ " +
+                " SELECT /*+ USE_NL(t o) LEADING(t o) INDEX(o pk_" + config.singleOverflowTable() + ") */ " +
                 "   o.id, o.val " +
-                " FROM pt_metropolis_overflow o, TABLE(CAST(? AS PT_MET_CELL_TS_TABLE)) t " +
+                " FROM " + config.singleOverflowTable() + " o, TABLE(CAST(? AS " + config.shared().tablePrefix() + "CELL_TS_TABLE)) t " +
```

Comment:
can we extract the creation of config.shared().tablePrefix() + "CELL_TS_TABLE" into it's own method.